### ISSUE TITLE
Consolidation of Glyph Correction Suggestions

### ISSUE THREAD

**Issue by kenlunde** [adobe-fonts/source-han-serif#39]
This issue is meant for tracking and submitting suggestions for glyph corrections. Issues that were submitted before this consolidation issue was opened are referenced by issue number.

**Comment by acuteaccent** [comment]
> Also to be adjusted are the glyphs _uni118C.vjmo01_, _uni1190.vjmo01_, _uni1192.vjmo01_, and _uni1112uni119Euni11D9_.

This should be separated into two, like this:

Also to be adjusted are the glyphs _uni118C.vjmo01_, _uni1190.vjmo01_, and _uni1192.vjmo01_.
Fix the glyph _uni1112uni119Euni11D9_.

**Comment by celestialphineas** [comment]
More about issue 14. The component 化 (U+5316) lack consistency.
![issue4](https://cloud.githubusercontent.com/assets/17267055/24937590/4f5e4bd6-1f64-11e7-92f3-927ab154bd67.PNG)

**Comment by hfhchan** [comment]
![image](https://cloud.githubusercontent.com/assets/8191296/25054665/0c2cb6e8-2191-11e7-8137-5122e1abc1f5.png)
U+6DEB: The middle stroke of the bottom left component for TW must be shorter than the bottom stroke, because 𡈼 (to grow) is the semantic component of U+6DEB (in excess), used instead of 壬.

**Comment by hfhchan** [comment]
![image](https://cloud.githubusercontent.com/assets/8191296/25054765/999837a0-2191-11e7-9a86-555b2fbb6e43.png)
U+5A6C is also affected.

**Comment by hfhchan** [comment]
![image](https://cloud.githubusercontent.com/assets/8191296/25069555/23cd3f58-22b7-11e7-9991-1c117ae46ae0.png)
A new glyph is needed for U+76E4 CN, it cannot share the same glyph with JP.

**Comment by hfhchan** [comment]
![image](https://cloud.githubusercontent.com/assets/8191296/25069573/bc335b10-22b7-11e7-8f59-58a193b38443.png)
CN glyph is wrong/unnecessary; it should be the same glyph as TW according to code charts:
![image](https://cloud.githubusercontent.com/assets/8191296/25069576/d0b39f00-22b7-11e7-92cb-91171c8eb7fc.png)

**Comment by kenlunde** [comment]
@hfhchan: U+76EC 盬 is best handled by swapping the glyphs for _uni76EC-CN_ and _uni76EC-TW_ (in my backend sources), then marking the latter (_uni76EC-TW_) for removal (see Issue #38).

**Comment by kenlunde** [comment]
U+76E4 盤 will be handled by adding a new CN glyph, _uni76E4-CN_ (see Issue #40).

**Comment by hfhchan** [comment]
![image](https://cloud.githubusercontent.com/assets/8191296/25074565/0272d03c-2331-11e7-8300-5b528bfe0c45.png)

U+7A07 稇: Second last stroke should be a dot instead of 捺.

**Comment by kenlunde** [comment]
Add CN glyph for U+7A07 稇, _uni7A07-CN_ (see Issue #40).

**Comment by hfhchan** [comment]
![image](https://cloud.githubusercontent.com/assets/8191296/25245400/d9223dea-2636-11e7-8079-409224886e36.png)
U+8285, the JP glyph is incorrect.

**Comment by kenlunde** [comment]
Nice catch. Thanks. Noted. Although this particular glyph is a candidate for removal for Version 2.000, we'll go ahead and fix it for the Version 1.000 dot-release.

**Comment by hfhchan** [comment]
For the whole lot of characters containing 廷, Source Han Serif has unified the shapes of the CN and TW in error. Excerpt:
![image](https://cloud.githubusercontent.com/assets/8191296/25270095/e40a637e-26b1-11e7-9fc8-79c6440e3f0b.png)

According to Tongyong Guifan Hanzi Biao, the second horizontal stroke should be the longest, while for MOE Standard Song, the second horizontal stroke should be the shortest.  The length of the second stroke was a point of contention as it was inconsistently printed as the shortest stroke in various standards preceding Tongyong Guifan Hanzi Biao.

Affected characters include at least U+5EF7, U+4FB9, U+5A17, U+5EAD, U+633A, U+6883, U+6D8F, U+70F6, U+73FD, U+7B73, U+7D8E, U+8121, U+8247, U+839B, U+8713, U+8A94, U+92CC, U+94E4, U+95AE, U+9706, U+9832, U+988B, U+9F2E, U+3E76, U+414D, U+4A60, U+4BD5 and U+4C53.

(As a side note, the Code Charts should probably be updated to use the fonts for Tongyong Guifan Hanzi Biao instead.)

**Comment by hfhchan** [comment]
![image](https://cloud.githubusercontent.com/assets/8191296/25273799/f9e1871e-26bf-11e7-869c-11e8e8263464.png)

For U+63EF, the second stroke of the right-most component of the CN, TW and KR glyph should be a slanted 豎, not 撇.

**Comment by hfhchan** [comment]
![image](https://cloud.githubusercontent.com/assets/8191296/25291405/72163c68-2704-11e7-8a3a-72a240015f44.png)
The dots for 35144 (U+83E1-TW) should be spread out like the JP glyph (35142)

**Comment by kenlunde** [comment]
@hfhchan: U+63EF 揯 will be handled by adding a new CN glyph, _uni63EF-CN_ (and its JP glyph, _uni63EF-JP_, is targeted for removal), and the TW glyph for U+83E1 菡, _uni83E1-TW_, will be adjusted.

**Comment by kenlunde** [comment]
@hfhchan: I am going through the CN and TW glyphs for the characters that include the shared 廷 component, and the current CN and TW glyphs for three of the 28 that you referenced above—U+7B73 筳, U+839B 莛, and U+9706 霆—appear to be okay as-is:

![three-glyphs](https://cloud.githubusercontent.com/assets/3988788/25304743/4d053c44-2722-11e7-9f81-dd76619e9235.jpg)

**Comment by hfhchan** [comment]
@kenlunde  yes these three examples are fine.  I must have copied the wrong string.

**Comment by kenlunde** [comment]
With regard to the other 25 characters, I finished my analysis and action plan.

The CN glyphs for the following 22 characters will be adjusted: U+3E76 㹶, U+414D 䅍, U+4A60 䩠, U+4BD5 䯕, U+4C53 䱓, U+5A17 娗, U+5EAD 庭, U+5EF7 廷, U+633A 挺, U+6883 梃, U+6D8F 涏, U+70F6 烶, U+73FD 珽, U+7D8E 綎, U+8121 脡, U+8247 艇, U+8713 蜓, U+8A94 誔, U+92CC 鋌, U+94E4 铤, U+95AE 閮, and U+9F2E 鼮.

New TW glyphs for the following two characters will be added: U+4FB9 侹 and U+9832 頲.

New TW glyphs for the following 10 characters will be added by using the current (Version 1.000) CN glyphs as-is: U+5EAD 庭, U+5EF7 廷, U+633A 挺, U+6883 梃, U+6D8F 涏, U+73FD 珽, U+7D8E 綎, U+8713 蜓, U+92CC 鋌, and U+95AE 閮.

The CN glyph for U+988B 颋, which is specific to CN, looks okay as-is.

**Comment by tamcy** [comment]
The strokes of 刀 component in 癩 U+7669 of JP/KR version (CID 28287) isn't aligned properly in Heavy weight.

![u 7669_28297](https://cloud.githubusercontent.com/assets/959433/25314214/5b74f6ec-2872-11e7-87a3-69050db16478.png)

**Comment by kenlunde** [comment]
@tamcy: Thank you for finding and reporting this. To clarify, this is an Adobe-designed JP glyph that corresponds to Adobe-Japan1-6 CID+5777, and the affected weights are SemiBold through Heavy.

**Comment by hfhchan** [comment]
![image](https://cloud.githubusercontent.com/assets/8191296/25324893/f75e66a0-28fa-11e7-9823-d0818acbd5ff.png)
TW glyph of U+750B: the second last dot should be completely joined to the side of 瓦 for "correctness" and consistency, compare with the other circled characters.

**Comment by tamcy** [comment]
![u 7d73_tw_el](https://cloud.githubusercontent.com/assets/959433/25341023/1ae0253e-293a-11e7-966e-616658c617dc.png)

Similar stroke alignment issue in U+7D73 絳 (CID 31785, TW variant, ExtraLight master)

**Comment by tamcy** [comment]
![tw_7f54_89a7](https://cloud.githubusercontent.com/assets/959433/25341457/6dda3468-293b-11e7-8046-9004f430aaf5.png)
Incorrect glyph for U+7F54 罔 (TW) and U+89A7(TW) 覽. Both requires a different glyph.
* Last stroke of 罔 should look the same as the one in KR. 
* The nineth stroke of 覽 should be the same as JP/KR which is a horizontal stroke.

**Comment by tamcy** [comment]
![u 8019_803b_tw](https://cloud.githubusercontent.com/assets/959433/25341659/1d60096c-293c-11e7-9878-d7c74c6d2cbd.png)

Incorrect mapping for U+8019 耙 and U+803B 耻 in TW. Both should look the same as the JP/KR glyph.

**Comment by kenlunde** [comment]
@hfhchan: The TW glyph for U+750B 甋 will be adjusted.

@tamcy: Thank you. The ExtraLight master of the TW glyph for U+7D73 絳 will be adjusted so that the stroke does not protrude. Also, the new TW glyph for U+7F54 罔 will be added. Lastly, U+89A7 覧 is in CNS 11643 Plane 3, and therefore outside the current scope of TW coverage. The good news is that an appropriate TW glyph for U+89A7 覧 will be included in Version 2.000 when HK support is added, and I will override the mappings accordingly.

**Comment by hfhchan** [comment]
![image](https://cloud.githubusercontent.com/assets/8191296/25420982/3b15a2b2-2a8d-11e7-9fa8-0ef80c70b724.png)
The KR glyph of U+8F44, U+5272, U+778E and U+8C41 may need adjustment; the fourth stroke on the right hand side should be a 撇 instead of 橫 according to the Code Charts.

![image](https://cloud.githubusercontent.com/assets/8191296/25421056/9443ca4e-2a8d-11e7-98d3-b59d4043ce9a.png)
![image](https://cloud.githubusercontent.com/assets/8191296/25421026/6f3d3fdc-2a8d-11e7-92e1-6132fd9df6e8.png)
![image](https://cloud.githubusercontent.com/assets/8191296/25421069/aad5a390-2a8d-11e7-8fb7-3ecc08decc1a.png)
![image](https://cloud.githubusercontent.com/assets/8191296/25421067/a71c1fae-2a8d-11e7-9ea8-49bd67551e1e.png)

--

If this issue is to be "corrected", then U+5BB3 needs a new glyph too:
![image](https://cloud.githubusercontent.com/assets/8191296/25421088/c6cf4858-2a8d-11e7-8d61-0ad3f5c58426.png)
![image](https://cloud.githubusercontent.com/assets/8191296/25421112/e46be5d8-2a8d-11e7-80f5-29e9fa4cf6bf.png)

**Comment by kenlunde** [comment]
@hfhchan: The KR glyphs for U+8F44 轄, U+5272 割, U+778E 瞎, U+8C41 豁, U+5BB3 害 (shared with JP) are intentional.

**Comment by tamcy** [comment]
![u8b04-tw](https://cloud.githubusercontent.com/assets/959433/25466981/43cacb88-2b3e-11e7-8261-57adcf6f8f77.png)

TW glyph of U+8B04 謄 (CID 38993) is incorrect - the 10th stroke should not pass through the 8th (horizontal) stroke.

It should be okay for TW to share the glyph of CN (CID 38992). However, the quality of the top-right component in CID 38992 looks subpar when compared with the other glyphs (just look at how the horizontal strokes are spaced), so it may also need to be adjusted.

**Comment by tamcy** [comment]
![u 8da3-tw](https://cloud.githubusercontent.com/assets/959433/25467236/8e3d177e-2b3f-11e7-87da-491b7e07d715.png)

TW glyph of U+8DA3 趣 (CID 40104) does not conform to MoE standard - the last stroke of 耳 won't penetrate the rightmost vertical stroke if it's not the rightmost component.

**Comment by tamcy** [comment]
![u91c5-tw](https://cloud.githubusercontent.com/assets/959433/25467334/128e312a-2b40-11e7-8e39-368f0d6db09b.png)

Similarly, the 耳-like component of TW glyph for U+91C5 釅 (42206) isn't correct.

**Comment by kenlunde** [comment]
@tamcy: Thank you. The issues with the TW glyphs for U+8B04 謄, U+8DA3 趣, and U+91C5 釅 will necessarily be dealt with after the Version 1.001 release.

**Comment by kenlunde** [comment]
@tamcy: Thank you. On second thought, it seems that we can squeeze these adjustments into the dot release:

A new TW glyph for U+8DA3 趣, _uni8DA3-TW_, will be added.

The TW glyph for U+8B04 謄, _uni8B04-TW_, will be marked for removal, though its glyph will be renamed to _uni8B04-CN_, then adjusted so that CN and TW can share the glyph (in principle, when CN and TW can share the same glyph, my workflow works best if the glyph is a CN one).

The TW glyph for U+91C5 釅, _uni91C5-TW_, will be fixed.

**Comment by kenlunde** [comment]
The image below includes the JP, CN, and TW ideographs that were corrected (also included are those glyphs that were redesigned):

![shserif-fixed-58](https://cloud.githubusercontent.com/assets/3988788/25528990/1711ba92-2bd5-11e7-89db-94d8697a028f.jpg)

**Comment by kenlunde** [comment]
Not shown above is the small number of adjustments and corrections to glyphs that correspond to Adobe-Japan1-6 kanji, six in total.

**Comment by tamcy** [comment]
![u5433-tw](https://cloud.githubusercontent.com/assets/959433/25906224/960d8334-35d6-11e7-8910-63d774f25d40.png)

Last stroke of 吳 (U+5433) is incorrect for TW glyph as per MoE's standard.

**Comment by kenlunde** [comment]
@tamcy: Noted in Issue #40 as a new TW glyph.

**Comment by jimmymasaru** [comment]
A new CN glyph for · _(U+00B7)_ is needed since a full-width form is usually preferred in this region and is a de facto standard.

**GB/T 15834―2011 标点符号用法 (Puncuation Marks Usage)**
<img width="579" alt="biaodian fuhao yongfa" src="https://user-images.githubusercontent.com/12769660/28244199-8747c66c-6a37-11e7-84b2-7fa3f8e5e481.png" />
PDF version can be found [here](https://github.com/Haixing-Hu/typesetting-standard/blob/master/数字文字/【GB:T%2015834-2011】标点符号用法.pdf).

[克里斯蒂娜·罗塞蒂 in Wikipedia](https://zh.wikipedia.org/wiki/克里斯蒂娜·罗塞蒂)
[一二·九运动 in Wikipedia](https://zh.wikipedia.org/wiki/一二·九运动)

**Comment by kenlunde** [comment]
@jimmymasaru: In lieu of adding a new full-width glyph for U+00B7, I suggest that I map U+00B7 to _uni30FB_ instead. Both characters have the same Unicode properties, so the double-mapping should work without problems.

**Comment by jimmymasaru** [comment]
@kenlunde That's sounds good to me. The only concern is locl feature since they are based on glyphs rather than unicode.
Besides, I'm thinking if we not only map _uni30FB_ to _U+00B7_ by default in CN version and also add a new substitution to change it to _uni00B7_ when the locl is English?

**Comment by kenlunde** [comment]
Right. Interaction the proportional version of U+00B7 via the 'locl' GSUB feature requires that U+00B7 will require its own full-width glyph, which can be a clone of the glyph for U+30FB. I just marked this for both families.

**Comment by tamcy** [comment]
For the TW/HK form of 畢, the "卄" component in the middle should be a disconnected "艹" as in 華. Following characters are affected:

嗶	U+55F6
彃	U+5F43
滭	U+6EED
熚	U+719A
畢	U+7562
篳	U+7BF3
縪	U+7E2A
罼	U+7F7C
蓽	U+84FD
蹕	U+8E55
鏎	U+93CE
驆	U+9A46
鷝	U+9DDD

(I hate to report this as I personally prefer the connected form 🤔)

**Comment by Explorer09** [comment]
@tamcy For 畢 component, the Taiwan MOE glyph references show the connected 卄.
But the 華 shows disconnected 十十

畢 (102648) http://language.moe.gov.tw/001/Upload/files/SITE_CONTENT/M0001/SUNGTI/as25.htm
華 (103503) http://language.moe.gov.tw/001/Upload/files/SITE_CONTENT/M0001/SUNGTI/as32.htm
篳 (203259) http://language.moe.gov.tw/001/Upload/files/SITE_CONTENT/M0001/SUNGTI/bs30.htm

**Comment by tamcy** [comment]
@Explorer09 You're right. I must have checked the wrong place. Turns out the disconnected form is for HK only. Sorry!

**Comment by opumps** [comment]
## **Thank you guys so much for this GREAT job!** 💯 
i use **source-han-serif 思源宋体** on every webpage in Chrome, and that's the only solution for me to solve the "SANS-alaways-in-Chinese-everywhere" problem. 
Also, **source-han-serif 思源宋体** provides a nice-looking Latin part, and that's why i gave up **STZHONGS华文中宋** the best of the best Chinese-Serif fonts. 
finally,  I can browse as comfortable as read the news paper. 


but, here come a little problem: 
![default](https://user-images.githubusercontent.com/10900982/33643594-0c64c8fa-da7b-11e7-8b3f-52e962738b1c.PNG)

>##  if you’ve seen this with any of Adobe’s other Asian fonts 
> ## if you've seen this with any of Adobe's other Asian fonts 

【’】and 【'】 are different, but the first one really gives me a  **:(**

**Comment by kenlunde** [comment]
@opumps For better or for worse, the best default for the single and double smart/curly quotes, at least for Chinese, are the full-width glyphs. These characters are problematic in that there is no ideal default, and I have a proposal for handling them via SVSes (_Standardized Variation Sequences_). BTW, this full-width default can also be found in our commercial Chinese fonts. Our Japanese fonts, to include the Japanese versions of _Source Han Sans_ and _Source Han Serif_, use proportional glyphs for these characters by default.

**Comment by Explorer09** [comment]
@opumps
I would argue that your use percentages are unsourced, besides it's not a sufficient argument for changing the default. Otherwise we might break the formatting of more contexts than what we have fixed.
I would suggest using language tagging as a workaround solution.

**Comment by Explorer09** [comment]
@kenlaude 
Sorry to bother again, but I wonder if it's possible to workaround the halfwidth/fullwidth quotes problem, by utilizing kerning or ligatures features in the font files? That is, have the curly quotes fullwidth by default in CN version, but kern them to proportional if the quotes precedes or follows a halfwidth Latin (or halfwidth punctuation)? Do you know what I mean?

**Comment by kenlunde** [comment]
While I prefer to stop the discussion about the smart/curly quotes in this issue, please see [L2/17-056](https://www.unicode.org/L2/L2017/17056-sv-western-vs-eastasian.pdf), which is my preliminary proposal to use SVSes to handle these and similar characters. If you have any meaningful or constructive feedback, please send it to me offline (aka via email).

**Comment by tamcy** [comment]
There is a bug in the heaviest master of the JP/KR glyph of U+5D1E（崞）. Intermediate weights are affected.
![5d1e](https://user-images.githubusercontent.com/959433/34413188-b8869160-ec1c-11e7-88e0-4499583f434f.png)

**Comment by kenlunde** [comment]
Noted, and thank you. I also noted that this particular glyph, _uni5D1E-JP_, is slated for removal in Version 2.000 in order to make room for HK glyphs.

**Comment by Man-Ting-Fang** [comment]
@jimmymasaru 

> A new CN glyph for · (U+00B7) is needed since a full-width form is usually preferred in this region and is a de facto standard.
> 
> GB/T 15834―2011 标点符号用法 (General Rules for Punctuation)

In Simplified Chinese, U+00B7 usually has a full-width form in practice; however, in theory it should be half-width according to GB/T 15834-2011, see the following figure. (Ironically, U+00B7 in GB/T 15834-2011 itself is NOT half-width.) Fortunately, GB/T 15834-2011 is NOT a compulsory standard ("GB/T" means that this is a recommendatory standard, T = 推荐 (Tuījiàn)).

![00](https://user-images.githubusercontent.com/17960358/34623395-381797ba-f28c-11e7-9a75-0db19bb66f24.PNG)

**Comment by kenlunde** [comment]
@Man-Ting-Fang: I have been thinking about U+00B7 recently, in my attempt to revise [L2/17-056](https://www.unicode.org/L2/L2017/17056-sv-western-vs-eastasian.pdf) by splitting it into two separate (and revised) proposals, and I think that the best way to address this issue for both the Simplified Chinese and Traditional Chinese fonts is to map U+00B7 to the glyph for U+30FB ・, _uni30FB_, which functions in a similar way, and I see no reason why the glyphs cannot be shared for the _Source Han_ families.

**Comment by tamcy** [comment]
The JP/KR glyph of 魢(U+9B62, ⿰魚己) looks the same as that of 𩵗 (U+29D57, ⿰魚巳). U+9B62 doesn't have a JP/KR source in the code chart. Not sure if this is an issue, I'm reporting it here anyway.

**Comment by kenlunde** [comment]
@tamcy: The good news is that the JP glyph for U+9B62 魢, _uni9B62-JP_, is slated for removal as part of the Version 2.000 update in order to make room for a large number of incoming HK glyphs, which means that this is likely to become a non-issue.

**Comment by extc** [comment]
I'd like to comment about the uni9FE6-TW. [英微] It should be 一儿 instead of 一几 for TW writing convention. I refer the IRGN1954R.pdf (page 16) from IRG42 website. Also,  from the source document provided from the said PDF file, I find the original image of the "New Testament in Chinese. (1864). (Archimandrite Gurias, Trans.) Beijing: Russian Mission." Retrieved from http://archive.wul.waseda.ac.jp/kosho/bunko08/bunko08_d0417/

[英微] should be like this: 
![image](https://user-images.githubusercontent.com/8214015/42392664-29fa9ce8-8186-11e8-83bf-4c4dfbe84d0d.png)

Please take reference the design of 兀 in the 微 of the above image, thank you.

**Comment by CNMan** [comment]
[GB/T 22321.1-2018 信息技术 中文编码字符集 汉字48点阵字型 第1部分：宋体](http://www.gb688.cn/bzgk/gb/newGbInfo?hcno=BCBF3BC7DCED3629F5E41CE02D9CFD55)
发布日期2018-06-07，实施日期2019-01-01

应该是中国大陆第一个包含CJK-C/D/E字符的字形标准

**Comment by kenlunde** [comment]
@CNMan I suspect that the Extension C, D, and E additions are to support 通用规范汉字表, which includes 44 Extension C characters, eight Extension D characters, and 108 Extension E characters. The _Source Han_ typefaces already support 通用规范汉字表 in its entirety.

**Comment by hfhchan** [comment]
I recommend against that standard, because the standard was published before the new GB18030 checking efforts, and so do not incorporate proper normalization of the C/D/E glyphs, nor corrections to existing URO/AB glyphs.

**Comment by tamcy** [comment]
卩 (U+5369) should be center-aligned (the glyph is inclined to the right, same for for Source Han Sans 1.x).

**Comment by kenlunde** [comment]
@tamcy Don't hold your breath for this one, especially for Version 2.000.

**Comment by kenlunde** [comment]
@tamcy To clarify, the glyph for U+5369 卩 should basically be identical to that of U+536A 卪, but without the center dot, right?

**Comment by tamcy** [comment]
Yes, this is what I meant, at least for HK. But after some digging I'm not very sure about the CN form. Here're some more information for your reference:

卩 U+5369 is a CJK Unified Ideographs block. In Source Han Sans/Serif it's regarded as an HK and probably CN character (not 100% sure about the latter). Here's what the [Unicode chart](https://unicode.org/charts/PDF/U4E00.pdf) shows:

![unicode](https://user-images.githubusercontent.com/959433/47093223-584b8000-d25b-11e8-8da7-60a65b7c3988.png)

And here's what the HKSCS document shows:

![hkscs](https://user-images.githubusercontent.com/959433/47093235-5d103400-d25b-11e8-88df-a8827e4ab936.png)

So I have confident that it should be center aligned, or its form should look like 卪 without the dot as you have said. This makes sense as 卪 and 卩 are essentially the ancient form of 節, so there's no reason for 卩 as a word to not align to the center. Also "⼙" as a radical is encoded differently (U+2F19, but it's also appear center-aligned in the [chart](https://unicode.org/charts/PDF/U2F00.pdf)).

But for the CN glyph, I do found the "right-aligned" form on the Unihan website and some other websites:

![unihan_zdic](https://user-images.githubusercontent.com/959433/47093312-7f09b680-d25b-11e8-9e73-b16bd4dbddb1.png)

I suppose the latest Unicode chart is authoritive?

**Comment by CNMan** [comment]
![2018-10-17_225240](https://user-images.githubusercontent.com/8692234/47095330-a1053800-d25f-11e8-8e02-48677a815fbf.png)
[中华人民共和国《信息技术 中文编码字符集》国家标准 GB 18030-2005](http://www.gb688.cn/bzgk/gb/newGbInfo?hcno=C344D8D120B341A8DD328954A9B27A99)

**Comment by kenlunde** [comment]
@CNMan I don't trust any one standard, and given that most standards, including older GB ones (a good example is GB/T 12345-1990), center the representative glyph for U+5369 卩, doing so makes the most sense. And, given that this glyph serves double-duty as a Kangxi Radical (U+2F19 ⼙), it makes for it to be centered in the em-box.

**Comment by kenlunde** [comment]
@extc With regard to the Traditional Chinese (TW and HK) glyph for U+9FE6 鿦, uni9FE6-HK, it will be good in _Source Han Sans_ Version 2.000. I made a note to fix the _Source Han Serif_ glyph in the next update, which is likely to be Version 2.000.

**Comment by lapomme** [comment]
![image](https://user-images.githubusercontent.com/27000939/47834359-a5f4dc00-dd75-11e8-92f2-bc1663702cff.png)
I think the JP/KR glyph for 这 would look better, and more consistent with other characters if it utilized a 長點 instead of a 捺 for the last stroke of 文. (Rough mockup at the bottom)

**Comment by kenlunde** [comment]
@lapomme This may or may not be good news, but this issue is likely to become a no-op, because the JP glyph for U+8FD9 这, _uni8FD9-JP_, is flagged for removal in Version 2.000, in order to make room for a large number of new HK glyphs.

**Comment by tamcy** [comment]
`uni5929-CN` (天): suggest to tune the heavy master so that the third stroke doesn't protrude the top of the first horizontal stroke.
![uni5929-CN](https://user-images.githubusercontent.com/959433/57358273-a26c6280-71a7-11e9-958c-f465f83c9fd7.png)

**Comment by stone-zeng** [comment]
`uni3514` (㔔): the points at the "circle" is not smooth:

![](https://user-images.githubusercontent.com/18233088/69005328-abf73400-095b-11ea-96e7-bffe09cd8fb0.png)

![](https://user-images.githubusercontent.com/18233088/69005316-78b4a500-095b-11ea-8e2f-60f868838121.png)

**Comment by KrasnayaPloshchad** [comment]
@tamcy @stone-zeng I’m shocked too for these problems, they looks just like how Rolls-Royce products the oil pipe to make engine explosion on Quantas Flight 32.
